Userland/Libraries/LibGUI/EmojiInputDialog.cpp

```diff
@@ -59,7 +59,7 @@ EmojiInputDialog::EmojiInputDialog(Window* parent_window)
     auto& scrollable_container = *main_widget->find_descendant_of_type_named<GUI::ScrollableContainerWidget>("scrollable_container"sv);
     m_search_box = main_widget->find_descendant_of_type_named<GUI::TextBox>("search_box"sv);
     m_toolbar = main_widget->find_descendant_of_type_named<GUI::Toolbar>("toolbar"sv);
-    m_emojis_widget = main_widget->find_descendant_of_type_named<GUI::Widget>("emojis"sv);
+    m_emojis_widget = scrollable_container.widget();
```

Comment:
Does `find_descendant_of_type_named` not work with the GML compiler?

Reply:
Yep, it does not, so I used a child widget as a workaround (in the above commit). [I had asked about it on the discord server.](https://discord.com/channels/830522505605283862/830739873119207426/1223644902223515658)

Reply:
I see, then that also explains the `EmojiInputDialog.gml` change that I was confused about. As long as everything still works as expected, then that's fine to me.